app/helpers/applications_helper.rb

```diff
@@ -0,0 +1,201 @@
+  end
+
+  def expert_reviewed_body
+    if current_user.sponsor?
+      'Congratulations! Your sponsorship application is complete and ready for submission.
+       Your SSP Pro Bono Lawyer can provide any support that you need with this process. '
+    else
+      'The expert file review for this application is complete.
+       Please connect with the sponsorship group to give them final information about submitting their file.'
+    end
+  end
+
+  def expert_reviewed_next_step(application)
+    return if current_user.sponsor?
+    link_to "Mark as submitted", mark_submitted_legal_internal_application_path(application), method: :put, class: "btn btn-success"
+  end
+
+  def submitted_heading
+    'Application submitted to GoC'
+  end
+
+  def submitted_body
+    if current_user.sponsor?
+      "Congratulations! We understand that your sponsorship application has been submitted to the Government of Canada.
+      Don't forget to let your SSP Pro Bono Lawyer know when you hear from Citizenship and Immigration Canada."
+    else
+      'This application has been submitted to the Government of Canada and is pending a response.
+       If you hear about the outcome of this application and it is positive, please use the relevant button to indicate this.
+       If it is rejected, please let us know and we will help advise you about next steps. '
+    end
+  end
+
+  def submitted_next_step(application)
+    return if current_user.sponsor?
+    link_to "Mark as accepted", mark_accepted_legal_internal_application_path(application), method: :put, class: "btn btn-success"
+  end
+
+  def accepted_heading
+    'Application accepted by GoC'
+  end
+
+  def accepted_body
+    if current_user.sponsor?
+      "Congratulations! We understand that your sponsorship application has been accepted and the refugees that you are sponsoring will be coming to Canada soon.
+       Don't forget that there are organizations in your area who can help with the settlement process.
+       The SSP would love to hear about your sponsorship story, the refugee(s) travel plans, and any other news that you have to share. Please don't hesitate to be in touch with us at refugeessp@uottawa.ca."
+    else
+      "Congratulations! We understand that this sponsorship application has been accepted and the refugees will be coming to Canada soon.
+       Don't forget to refer your clients to the organizations in your area who can help with the settlement process (beyond SSP scope).
+       The impact that your assistance as an SSP Pro Bono Lawyer has made cannot be understated.
+       Thank you for giving your time and skill to this sponsorship group through the SSP.
+       The SSP would love to hear about your experience as an SSP Pro Bono Lawyer and any other news that you have to share.
+       We would also love to facilitate another match. Please don't hesitate to be in touch with us at refugeessp@uottawa.ca."
+     end
+  end
+
+  def accepted_next_step(application)
```

Comment:
Unused method argument - `application`. If it's necessary, use `_` or `_application` as an argument name to indicate that it won't be used. You can also write as `accepted_next_step(*)` if you want the method to accept any arguments but don't care about them.